apps/admin-portal/src/pages/application-template.tsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from "react";
+import { Grid, Icon, Input } from "semantic-ui-react";
+import { ApplicationTemplate } from "../components";
+import { history } from "../helpers/history";
+import { PageLayout } from "../layouts";
+
+/**
+ * Choose the application template from this page.
+ *
+ * @return {JSX.Element}
+ */
+export const ApplicationTemplateSelectPage = (): JSX.Element => {
+
+    // Template list
```

Comment:
Need to localize this. Add a `TODO:` if you planing to do later

Reply:
Templates will be retrieved from the backend in the future.